src/components/FlatListItem/index.js

```diff
@@ -39,7 +39,7 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     paddingVertical: 16,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.lightgrey` to `COLORS.gray100` for the borderBottomColor property seems fine. It's a simple style change and should not affect the functionality of the component.
```diff
-    borderBottomColor: COLORS.lightgrey,
+    borderBottomColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->